www/src/views/showcase/filtered-showcase.js

```diff
@@ -0,0 +1,476 @@
+import React, { Component } from "react"
+import MdClear from "react-icons/lib/md/clear"
+import MdCheckboxBlank from "react-icons/lib/md/check-box-outline-blank"
+import MdCheckbox from "react-icons/lib/md/check-box"
+import MdArrowDownward from "react-icons/lib/md/arrow-downward"
+import MdFilterList from "react-icons/lib/md/filter-list"
+import Fuse from "fuse.js"
+import hex2rgba from "hex2rgba"
+import { style } from "glamor"
+
+import ShowcaseList from "./showcase-list"
+import Collapsible from "./collabsible"
```

Comment:
typo

Reply:
oh good catch, is that the filename as well, how is that working then?

Reply:
yea you matched your typo with another typo haha